plugins/modules/pacman_key.py

```diff
@@ -129,12 +129,55 @@
 from ansible.module_utils.common.text.converters import to_native
 
 
+class GpgListResult:
```

Comment:
We still support Python 2.7.
```suggestion
class GpgListResult(object):
```